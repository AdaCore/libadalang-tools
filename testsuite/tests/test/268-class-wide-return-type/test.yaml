description: |
  Test if private tagged types as return type are accepted and test cases can
  be generated

driver: gnattest_tgen
default-gpr: true
