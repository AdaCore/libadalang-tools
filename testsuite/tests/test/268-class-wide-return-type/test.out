pkg.ads:5:4: error: corresponding test FAILED: Test not implemented. (pkg-test_data-tests.adb:61)
1 tests run: 0 passed; 1 failed; 0 crashed.
Test runner dumped 0 tests
